pages/print.js

```diff
@@ -41,6 +45,13 @@ const styles = () => ({
   svgContainer: {
     width: "320px",
     height: "30px"
+  },
+  hr: {
+    color: "black",
+    backgroundColor: "black",
+    border: 0,
+    height: "2px",
+    "-webkit-print-color-adjust": "exact"
```

Comment:
We want to print background images? Is this for the lines?

Reply:
Yeah part of the `<hr>` line colour is actually a background colour. So this is to make sure it gets printed.